features/geometrisch-zoeken.feature

```diff
@@ -0,0 +1,136 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik met een geometrie (punt, bounding box) kunnen zoeken naar resources, 
+                 zodat ik gericht vervolgstappen kan definiëren m.b.t. de gevonden resources.
+
```

Comment:
is dit niet gewoon zoeken met bounding box? 
Je beschrijft hieronder niet WAT er onder welke condities geleverd wordt, maar HOE (op welke manier en met welke hulpmiddelen) dit wordt geleverd. Wij willen heel graag proberen om de functionaliteit eenduidig te beschrijven, en indien mogelijk deze gebruiken om te testen. Misschien even over hebben?